From 6f8d0ca068adf4dfff517677d5315293bd4f9800 Mon Sep 17 00:00:00 2001
From: falkTX <falktx@gmail.com>
Date: Mon, 11 Apr 2016 12:16:51 +0200
Subject: [PATCH 0896/1936] Fix build using latest cmake and buildroot I tested
 it locally with an old version too, which works. Let me know if this causes
 any issues.

---
 src/Plugin/AlienWah/CMakeLists.txt      | 2 +-
 src/Plugin/Chorus/CMakeLists.txt        | 2 +-
 src/Plugin/Distortion/CMakeLists.txt    | 2 +-
 src/Plugin/DynamicFilter/CMakeLists.txt | 2 +-
 src/Plugin/Echo/CMakeLists.txt          | 2 +-
 src/Plugin/Phaser/CMakeLists.txt        | 2 +-
 src/Plugin/Reverb/CMakeLists.txt        | 2 +-
 src/Plugin/ZynAddSubFX/CMakeLists.txt   | 2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/Plugin/AlienWah/CMakeLists.txt b/src/Plugin/AlienWah/CMakeLists.txt
index 40e67c6a..8c650881 100644
--- a/src/Plugin/AlienWah/CMakeLists.txt
+++ b/src/Plugin/AlienWah/CMakeLists.txt
@@ -21,7 +21,7 @@ install(TARGETS ZynAlienWah_lv2 LIBRARY DESTINATION ${PluginLibDir}/lv2/ZynAlien
 install(TARGETS ZynAlienWah_vst LIBRARY DESTINATION ${PluginLibDir}/vst/)
 
 add_custom_command(TARGET ZynAlienWah_lv2 POST_BUILD
-    COMMAND lv2-ttl-generator $<TARGET_FILE:ZynAlienWah_lv2>
+    COMMAND ../../lv2-ttl-generator $<TARGET_FILE:ZynAlienWah_lv2>
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lv2)
 
 install(FILES
diff --git a/src/Plugin/Chorus/CMakeLists.txt b/src/Plugin/Chorus/CMakeLists.txt
index ba538b9c..a9dbe038 100644
--- a/src/Plugin/Chorus/CMakeLists.txt
+++ b/src/Plugin/Chorus/CMakeLists.txt
@@ -21,7 +21,7 @@ install(TARGETS ZynChorus_lv2 LIBRARY DESTINATION ${PluginLibDir}/lv2/ZynChorus.
 install(TARGETS ZynChorus_vst LIBRARY DESTINATION ${PluginLibDir}/vst/)
 
 add_custom_command(TARGET ZynChorus_lv2 POST_BUILD
-    COMMAND lv2-ttl-generator $<TARGET_FILE:ZynChorus_lv2>
+    COMMAND ../../lv2-ttl-generator $<TARGET_FILE:ZynChorus_lv2>
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lv2)
 
 install(FILES
diff --git a/src/Plugin/Distortion/CMakeLists.txt b/src/Plugin/Distortion/CMakeLists.txt
index cfa9fd24..0a0affa6 100644
--- a/src/Plugin/Distortion/CMakeLists.txt
+++ b/src/Plugin/Distortion/CMakeLists.txt
@@ -21,7 +21,7 @@ install(TARGETS ZynDistortion_lv2 LIBRARY DESTINATION ${PluginLibDir}/lv2/ZynDis
 install(TARGETS ZynDistortion_vst LIBRARY DESTINATION ${PluginLibDir}/vst/)
 
 add_custom_command(TARGET ZynDistortion_lv2 POST_BUILD
-    COMMAND lv2-ttl-generator $<TARGET_FILE:ZynDistortion_lv2>
+    COMMAND ../../lv2-ttl-generator $<TARGET_FILE:ZynDistortion_lv2>
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lv2)
 
 install(FILES
diff --git a/src/Plugin/DynamicFilter/CMakeLists.txt b/src/Plugin/DynamicFilter/CMakeLists.txt
index e3e89f52..0fd8ed78 100644
--- a/src/Plugin/DynamicFilter/CMakeLists.txt
+++ b/src/Plugin/DynamicFilter/CMakeLists.txt
@@ -21,7 +21,7 @@ install(TARGETS ZynDynamicFilter_lv2 LIBRARY DESTINATION ${PluginLibDir}/lv2/Zyn
 install(TARGETS ZynDynamicFilter_vst LIBRARY DESTINATION ${PluginLibDir}/vst/)
 
 add_custom_command(TARGET ZynDynamicFilter_lv2 POST_BUILD
-    COMMAND lv2-ttl-generator $<TARGET_FILE:ZynDynamicFilter_lv2>
+    COMMAND ../../lv2-ttl-generator $<TARGET_FILE:ZynDynamicFilter_lv2>
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lv2)
 
 install(FILES
diff --git a/src/Plugin/Echo/CMakeLists.txt b/src/Plugin/Echo/CMakeLists.txt
index fe1ad015..fe4e47ee 100644
--- a/src/Plugin/Echo/CMakeLists.txt
+++ b/src/Plugin/Echo/CMakeLists.txt
@@ -21,7 +21,7 @@ install(TARGETS ZynEcho_lv2 LIBRARY DESTINATION ${PluginLibDir}/lv2/ZynEcho.lv2/
 install(TARGETS ZynEcho_vst LIBRARY DESTINATION ${PluginLibDir}/vst/)
 
 add_custom_command(TARGET ZynEcho_lv2 POST_BUILD
-    COMMAND lv2-ttl-generator $<TARGET_FILE:ZynEcho_lv2>
+    COMMAND ../../lv2-ttl-generator $<TARGET_FILE:ZynEcho_lv2>
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lv2)
 
 install(FILES
diff --git a/src/Plugin/Phaser/CMakeLists.txt b/src/Plugin/Phaser/CMakeLists.txt
index 99262bb2..ce4a1cfa 100644
--- a/src/Plugin/Phaser/CMakeLists.txt
+++ b/src/Plugin/Phaser/CMakeLists.txt
@@ -21,7 +21,7 @@ install(TARGETS ZynPhaser_lv2 LIBRARY DESTINATION ${PluginLibDir}/lv2/ZynPhaser.
 install(TARGETS ZynPhaser_vst LIBRARY DESTINATION ${PluginLibDir}/vst/)
 
 add_custom_command(TARGET ZynPhaser_lv2 POST_BUILD
-    COMMAND lv2-ttl-generator $<TARGET_FILE:ZynPhaser_lv2>
+    COMMAND ../../lv2-ttl-generator $<TARGET_FILE:ZynPhaser_lv2>
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lv2)
 
 install(FILES
diff --git a/src/Plugin/Reverb/CMakeLists.txt b/src/Plugin/Reverb/CMakeLists.txt
index a38f5b61..b61564d2 100644
--- a/src/Plugin/Reverb/CMakeLists.txt
+++ b/src/Plugin/Reverb/CMakeLists.txt
@@ -21,7 +21,7 @@ install(TARGETS ZynReverb_lv2 LIBRARY DESTINATION ${PluginLibDir}/lv2/ZynReverb.
 install(TARGETS ZynReverb_vst LIBRARY DESTINATION ${PluginLibDir}/vst/)
 
 add_custom_command(TARGET ZynReverb_lv2 POST_BUILD
-    COMMAND lv2-ttl-generator $<TARGET_FILE:ZynReverb_lv2>
+    COMMAND ../../lv2-ttl-generator $<TARGET_FILE:ZynReverb_lv2>
 	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lv2)
 
 install(FILES
diff --git a/src/Plugin/ZynAddSubFX/CMakeLists.txt b/src/Plugin/ZynAddSubFX/CMakeLists.txt
index bc8e905f..41ff3e5a 100644
--- a/src/Plugin/ZynAddSubFX/CMakeLists.txt
+++ b/src/Plugin/ZynAddSubFX/CMakeLists.txt
@@ -76,7 +76,7 @@ install(TARGETS ZynAddSubFX_lv2 LIBRARY DESTINATION ${PluginLibDir}/lv2/ZynAddSu
 install(TARGETS ZynAddSubFX_vst LIBRARY DESTINATION ${PluginLibDir}/vst/)
 
 add_custom_command(TARGET ZynAddSubFX_lv2 POST_BUILD
-    COMMAND lv2-ttl-generator $<TARGET_FILE:ZynAddSubFX_lv2>
+    COMMAND ../../lv2-ttl-generator $<TARGET_FILE:ZynAddSubFX_lv2>
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lv2)
 
 install(FILES
-- 
2.47.0

